Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/SurveyConfigMapper.kt

```diff
@@ -5,17 +5,46 @@ import de.rki.coronawarnapp.appconfig.SafetyNetRequirements
 import de.rki.coronawarnapp.appconfig.SafetyNetRequirementsContainer
 import de.rki.coronawarnapp.appconfig.SurveyConfig
 import de.rki.coronawarnapp.server.protocols.internal.v2.AppConfigAndroid
+import timber.log.Timber
 import javax.inject.Inject
 
 @Reusable
 class SurveyConfigMapper @Inject constructor() : SurveyConfig.Mapper {
     override fun map(rawConfig: AppConfigAndroid.ApplicationConfigurationAndroid): SurveyConfig {
-        // TODO
-        return SurveyConfigContainer()
+        var surveyConfigContainer = SurveyConfigContainer()
+
+        if (rawConfig.hasEventDrivenUserSurveyParameters()) {
+            if (rawConfig.eventDrivenUserSurveyParameters.hasCommon()) {
+                rawConfig.eventDrivenUserSurveyParameters.common.also {
+                    surveyConfigContainer = SurveyConfigContainer(
+                        otpQueryParameterName = it.otpQueryParameterName,
+                        surveyOnHighRiskEnabled = it.surveyOnHighRiskEnabled,
+                        surveyOnHighRiskUrl = it.surveyOnHighRiskUrl
```

Comment:
Should we validate these? e.g. "isNotEmpty()"?

Reply:
And should we also validate if the url-is well-formed here? Currently, I am doing this in the `SurveyUrlProvider` component of my subtask, but if we validate it here and I can be sure to have a valid URL, I don't need to do that anymore. 

What about exposing `surveyOnHighRiskUrl` as a `okhttp3.HttpUrl` instead of `String` in `SurveyConfigContainer`?

Reply:
@d4rken I already use the default values as fallback (see skelton)
@LukasLechnerDev will adjust to validate url here